### PR TITLE
Add more diagrams to retry design

### PR DESCRIPTION
**Pull request by ncteisen**

### CODE REVIEW

File: A6-client-retries.md [hunk 1]
```diff
@@ -262,6 +274,10 @@ Similarly, if the RPC reaches the server, but has never been seen by the server
 
 Since retry throttling is designed to prevent server application overload, and these transparent retries do not make it to the server application layer, they do not count as failures when deciding whether to throttle retry attempts.
 
+![State Diagram](A6_graphics/transparent.png)
```
Comment (ericgribkoff) [inline A6-client-retries.md]: Diagram caption changes:

s/Failure occurs in the before/Failure occurs before/
s/max_attempts/maxRetryAttempts/

File: A6-client-retries.md [hunk 2]
```diff
@@ -96,6 +100,10 @@ Each of these configuration options is detailed in its own section below.
 
 gRPC's call deadlines apply to the entire chain of original RPC and retry attempts. The time limit begins when the original RPC is issued to the gRPC client library. For example, if the specified deadline is 500ms and after 500ms only one retry has been attempted, the call is cancelled even if `maxRetryAttempts` is set to two or greater.
 
+![State Diagram](A6_graphics/too_many_attempts.png)
```
Comment (ericgribkoff) [inline A6-client-retries.md]: This diagram is for maxRetryAttempts=2, right? Maybe add this to the image directly or here in the .md file, as our other examples have maxRetryAttempts=3.

Comment (ncteisen) [reply A6-client-retries.md]: I changed my diagram to 3 attempts

File: A6-client-retries.md (hunk 1 above)
Comment (ncteisen) [reply A6-client-retries.md]: Done

**Pull request merged**